src/test/java/com/gravity9/mongocdc/ConfigManagerTest.java

```diff
@@ -1,27 +1,25 @@
 package com.gravity9.mongocdc;
 
-import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
-import static com.gravity9.mongocdc.constants.TestConstants.COLL_NAME;
-import static com.gravity9.mongocdc.constants.TestConstants.CONN_URI;
-import static com.gravity9.mongocdc.constants.TestConstants.DB_NAME;
 import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
-public class ConfigManagerTest {
+public class ConfigManagerTest extends AbstractMongoDbBase {
 
-	private MongoConfig.MongoConfigBuilder mongoConfigBuilder = new MongoConfig.MongoConfigBuilder()
-			.connectionUri("mongodb://localhost:27017,localhost:27018,localhost:27019/?replicaSet=dbrs&retryWrites=true&w=majority")
-			.databaseName("test-db")
-			.collectionName("example")
-			.workerConfigCollectionName("changeStreamWorkerConfig")
-			.clusterConfigCollectionName("changeStreamClusterConfig");
+	private MongoConfig.MongoConfigBuilder mongoConfigBuilder;
 
-	@AfterEach
-	void cleanUp() {
-		TestMongoUtils.cleanUp();
+	@BeforeEach
+	public void setup() {
+		super.setup();
+		mongoConfigBuilder = new MongoConfig.MongoConfigBuilder()
+				.connectionUri(getConnectionUri())
+				.databaseName(getDatabaseName())
+				.collectionName(getTestCollectionName())
+				.workerConfigCollectionName("changeStreamWorkerConfig")
```

Comment:
the same as in previous comment - use methods from AbstractMongoDbBase

Reply:
Fixed